tests/functional/test_zilien_reader.py

```diff
@@ -3,20 +3,31 @@
 import datetime
 import time
 from collections import namedtuple
+from functools import reduce
 from pathlib import Path
 
+import numpy as np
 import pytest
+from pytest import approx, fixture
+
 from ixdat import Measurement
-from pytest import approx
 from ixdat.techniques.ec_ms import (
     ECMSMeasurement,
     MSMeasurement,
     ECMeasurement,
 )
 
-DATA_DIR = Path(__file__).parent.parent.parent / "test_data"
 
-PATH_TO_DATAFILE = DATA_DIR / "Zilien version 1" / "2022-04-06 16_17_23 full set.tsv"
+DIR_VERSION_1 = (
+    Path(__file__).parent.parent.parent
+    / "submodules/ixdat-large-test-files/zilien_version_1"
```

Comment:
I don't see these in the ixdat-large-test-files repository. Please make that PR.
Looking at it here now: https://github.com/matenestor/ixdat-large-test-files

Reply:
The PR is already opened here https://github.com/ixdat/ixdat-large-test-files/pull/3.